client/src/panels/panel_declarations/finances/transfer_payments/historical_g_and_c.js

```diff
@@ -179,6 +179,7 @@ class DetailedHistTPItems extends React.Component {
     
     const raw_data = _.flatMap(graph_series, value => value);
 
+    const all_tp_idx = _.map(prepped_rows, (row, idx) => idx);
```

Comment:
There's a lodash util called `range` that would give the same result by doing `_.range(prepped_rows.length)`, just a tip.

Reply:
Ah, never knew this util existed. Noted, thanks!